starfish/test/full_pipelines/api/test_merfish.py

```diff
@@ -232,8 +232,8 @@ def test_merfish_pipeline_cropped_data():
 
     # assert that number of high-expression detected genes are correct
     expected_counts = pd.Series(
-        [120, 101, 54, 43, 30],
-        index=('nan', 'MALAT1', 'SRRM2', 'FASN', 'PRKDC')
+        [104, 63, 55, 45, 32],
+        index=('MALAT1', 'nan', 'SRRM2', 'FASN', 'TLN1')
```

Comment:
why is 'nan' still showing up? it's certainly reduced by ~50% but it should still not appear as long as these spots pass thresholds. right?